watertap/unit_models/pressure_exchanger.py

```diff
@@ -208,8 +221,12 @@ def eq_pressure_transfer(b, t):
             self.flowsheet().config.time,
             doc="Equal volumetric flow rate")
         def eq_equal_flow_vol(b, t):
-            return (b.high_pressure_side.properties_in[t].flow_vol ==
-                    b.low_pressure_side.properties_in[t].flow_vol)
+            if self.config.has_mass_transfer:
+                return (b.high_pressure_side.properties_out[t].flow_vol ==
+                        b.low_pressure_side.properties_in[t].flow_vol)
+            else:
+                return (b.high_pressure_side.properties_in[t].flow_vol ==
+                        b.low_pressure_side.properties_in[t].flow_vol)
```

Comment:
I think this conditional can be removed. The relationship for when there is mass transfer is still applicable for when there isn't mass transfer.

Reply:
Will do. That make sense.